components/transfer/__tests__/__snapshots__/demo-extend.test.ts.snap

```diff
@@ -6539,27 +6539,7 @@ Array [
                           <tr>
                             <th
                               class="ant-table-cell ant-table-selection-column"
-                            >
-                              <div
-                                class="ant-table-selection"
-                              >
-                                <label
-                                  class="ant-checkbox-wrapper"
-                                >
-                                  <span
-                                    class="ant-checkbox"
-                                  >
-                                    <input
-                                      class="ant-checkbox-input"
-                                      type="checkbox"
-                                    />
-                                    <span
-                                      class="ant-checkbox-inner"
-                                    />
-                                  </span>
-                                </label>
-                              </div>
-                            </th>
```

Comment:
Should not change it, please check failed test cases.

Reply:
ok

Reply:
done